kitsune/users/auth.py

```diff
@@ -210,6 +211,13 @@ def update_user(self, user, claims):
             if user_attr_changed:
                 user.save()
             profile.save()
+
+        # If we have an updated email, let's update Zendesk too
+        # the check is repeated for now but it will save a few
+        # API calls if we trigger the task only when we know that we have new emails
+        if user_attr_changed:
+            update_zendesk_identity.delay(user.id, email)
```

Comment:
Though this will work, I'm not a fan of tying zendesk profile updates directly to fxa updates. IMO fxa should trigger a django profile update, and django profile updates should trigger a zendesk update.

Reply:
We discussed this more and we are going to keep the updates in this method